content/en/tracing/setup/nginx.md

```diff
@@ -96,6 +99,34 @@ Complete examples:
 
 After completing this configuration, HTTP requests to NGINX will initiate and propagate Datadog traces, and will appear in the APM UI.
 
+## NGINX Ingress Controller
+
+The Kubernetes [ingress-nginx](https://github.com/kubernetes/ingress-nginx) controller versions 0.23.0+ include the NGINX plugin for OpenTracing.
+
+To enable this plugin, create or edit a ConfigMap to set `enable-opentracing: "true"` and the `datadog-collector-host` to which traces should be sent.
+The name of the ConfigMap will be cited explicitly by the nginx-ingress controller container's command line argument, defaulting to `--configmap=$(POD_NAMESPACE)/nginx-configuration`.
+If ingress-nginx was installed via helm chart, this ConfigMap will be named like `Release-Name-nginx-ingress-controller`.
+
+The ingress controller manages both the `nginx.conf` and `/etc/nginx/opentracing.json` files. Tracing is enabled for all Locations.
```

Comment:
Does it make sense to replace "Locations" with "`location` sections" or "`location` blocks"?

Reply:
Yes. I believe "block" is the correct term for the `{inside}` of an NGINX "directive". I will `s/section/block/` in the added paragraph _and_ earlier in the same file.